composer.json

```diff
@@ -30,7 +30,8 @@
     "d11wtq/boris": "~1.0",
     "symfony/yaml": "~2.2",
     "symfony/var-dumper": "2.6.3",
-    "pear/console_table": "~1.2.0"
+    "pear/console_table": "~1.2.0",
+    "drupal/console": "~0"
```

Comment:
If Drupal Console is required, then it gives us a mechanism to use its code to start migrating Drush commands to the Symfony console.  At the moment, though, this PR is just a proof-of-concept.  At first, there won't be any Drush commands that require Drupal Console, so we could certainly start off with this as a recommended package.